### PR TITLE
iSpringSolutionsInc.FreeCam8 version 8.7.27159

### PR DESCRIPTION
**Pull request by SamCurant453**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16398)

### CODE REVIEW

File: manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: iSpringSolutionsInc.FreeCam8
```
Comment (ItzLevvie) [inline manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.yaml:3]:
```suggestion
PackageIdentifier: iSpringSolutions.FreeCam8
```

File: manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: iSpringSolutionsInc.FreeCam8
+PackageVersion: 8.7.27159
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (ItzLevvie) [inline manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml:7]:
```suggestion
  Architecture: x86
```
x86 will still install on both x86 and x64 devices.

File: manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: iSpringSolutionsInc.FreeCam8
```
Comment (ItzLevvie) [inline manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml:3]:
```suggestion
PackageIdentifier: iSpringSolutions.FreeCam8
```

File: manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: iSpringSolutionsInc.FreeCam8
```
Comment (ItzLevvie) [inline manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: iSpringSolutions.FreeCam8
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

You'll need to rename the folder from `iSpringSolutionsInc` to `iSpringSolutions` using the GitHub Desktop client on Windows.

File: manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml (hunk 2 above)
Comment (SamCurant453) [reply manifests/i/iSpringSolutionsInc/FreeCam8/8.7.27159/iSpringSolutionsInc.FreeCam8.installer.yaml:7]: hmm

**Comment by SamCurant453** [comment]
can anybody tell? why this is showing manifest validation error and metadata not compliant??

> Hello @SamCurant453,
> The package manager bot determined that the metadata was not compliant.
> 
> Please verify the manifest file is compliant with the package manager [1.0 manifest specification](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md).
> Make sure the ID is of the form publisher.appname and that the folder structure is manifests\partition\publisher\appname\version.
> Note: The path and "PackageIdentifier" are case sensitive.
> Be sure to use a tool like VSCode (https://code.visualstudio.com/) to make sure the manifest YAML syntax is correct.
> 
> You could also try our [Windows Package Manager Manifest Creator Preview](https://github.com/microsoft/winget-create).
> 
> For details on the specific error, see the details link below in the build pipeline.

**Pull request closed without merging**